src/connection.h

```diff
@@ -31,6 +31,8 @@
 #ifndef __REDIS_CONNECTION_H
 #define __REDIS_CONNECTION_H
 
+#include <sys/errno.h>
```

Comment:
Look like this had a warning, i modify it in https://github.com/redis/redis/pull/9739

https://github.com/redis/redis/runs/4111889932?check_suite_focus=true#step:5:494